Create script to compile sandboxes

pvollan opened the pull request:
#### 4ebd1e23ed02ff1181b6479b929ccf94b2fc2fe9
<pre>
Create script to compile sandboxes
<a href="https://bugs.webkit.org/show_bug.cgi?id=264093">https://bugs.webkit.org/show_bug.cgi?id=264093</a>
<a href="https://rdar.apple.com/117854457">rdar://117854457</a>

Reviewed by Elliott Williams.

* Source/WebKit/DerivedSources.make:
* Source/WebKit/Scripts/compile-sandbox.sh: Added.

Canonical link: <a href="https://commits.webkit.org/270385@main">https://commits.webkit.org/270385@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/059f6c782df9a1c7e356494fd00296cee3693368

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/25307 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/3848 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/26562 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/27419 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/23212 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/5566 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/1281 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/23415 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/25549 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/2856 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/21834 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/27997 "Built successfully") | 
| [✅ 🧪 webkitpy](https://ews-build.webkit.org/#/builders/6/builds/25467 "Passed tests") | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/2532 "Passed tests") | | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/28880 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/23082 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/23127 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/26718 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/2487 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/771 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/3843 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/6076 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/2930 "Built successfully") | | [⏳ 🛠 jsc-mips ](https://ews-build.webkit.org/#/builders/JSC-MIPSEL-32bits-Build-EWS "Waiting in queue, processing has not started yet") | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/2822 "Built successfully") | | [⏳ 🧪 jsc-mips-tests ](https://ews-build.webkit.org/#/builders/JSC-MIPSEL-32bits-Build-EWS "Waiting in queue, processing has not started yet") | 
<!--EWS-Status-Bubble-End-->

emw-apple commented on Source/WebKit/Scripts/compile-sandbox.sh:
```diff
@@ -0,0 +1,39 @@
+
+SANDBOX_PATH=$1;
+SANDBOX_NAME=$2;
+SDK_NAME=$3;
```
nit: This will be in the environment already, so you don't need to pass it in.

emw-apple commented on Source/WebKit/Scripts/compile-sandbox.sh:
```diff
@@ -0,0 +1,39 @@
+    fi
+fi;
+
+exit 0;
```
nit: `exit 0` at the end of the script is superfluous

pvollan replied:
I have addressed these issues in the latest patch.

Thanks for reviewing!

emw-apple reviewed (approved):
Seems reasonable, thank you! Couple minor bits of feedback, but I'm comfortable giving this my r+ now.

pvollan committed:
Create script to compile sandboxes
https://bugs.webkit.org/show_bug.cgi?id=264093
rdar://117854457

Reviewed by Elliott Williams.

* Source/WebKit/DerivedSources.make:
* Source/WebKit/Scripts/compile-sandbox.sh: Added.

Canonical link: https://commits.webkit.org/270385@main
